TipCatDotNet.Api/Models/HospitalityFacilities/PaymentDetailsResponse.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace TipCatDotNet.Api.Models.HospitalityFacilities
+{
+    public readonly struct PaymentDetailsResponse
+    {
+        public PaymentDetailsResponse(int receiverId, string receiverFirstName, string receiverLastName, string? receiverAvatarUrl)
+        {
+            ReceiverId = receiverId;
+            ReceiverFirstName = receiverFirstName;
+            ReceiverLastName = receiverLastName;
+            ReceiverAvatarUrl = receiverAvatarUrl;
+        }
+
+
+        [Required]
+        public int ReceiverId { get; }
```

Comment:
Probably, we should rename `receiver` to `member`, because we have two names for one entity now